src/water_quality/cli/process_annual_wq_variables.py

```diff
@@ -249,44 +239,54 @@ def cli(
                 sigma_coefficient=SC,
             )
 
-            # Dark pixel correction
-            ds = R_correction(ds, instruments_to_use, WFTL)
+            # Floating Algea Index
+            ds = geomedian_FAI(ds)
 
-            if "msi_agm" in instruments_list.keys():
-                log.info("Calculating the hue.")
-                ds["hue"] = hue_calculation(ds, instrument="msi_agm")
+            # NDVI
+            ds = geomedian_NDVI(ds)
 
-            if "msi_agm" in instruments_list.keys():
-                log.info(
-                    "Determining the open water type for each pixel "
-                    "using the instrument msi_agm"
-                )
-                ds["owt_msi"] = OWT_pixel(
-                    ds,
-                    instrument="msi_agm",
-                    resample_rate=3,
-                )
+            # Set the clear water mask
+            ds["clearwater"] = xr.where(
+                np.isnan(ds.agm_fai),
+                xr.where(ds.water_mask == 1, True, False),
+                False,
+            )
 
-            if "oli_agm" in instruments_list.keys():
-                log.info(
-                    "Determining the open water type for each pixel "
-                    "using the instrument oli_agm"
-                )
-                ds["owt_oli"] = OWT_pixel(
-                    ds,
-                    instrument="oli_agm",
-                    resample_rate=3,
-                )
+            # Reflectance correction
+            ds = R_correction(ds, instruments_to_use, drop=False)
 
-            log.info("Applying the WQ algorithms to water areas.")
+            # Hue calculation
+            ds = geomedian_hue(ds)
+
+            # OWT calculation
+            # Turned off OWT as I work on fix
+            # TODO: turn on once owt_module is fixed
+            # OWT calculation
+            # ds = run_OWT(ds)
+
+            # Mask dataset based on water frequency threshold
+            mask = (ds.wofs_ann_freq >= WFTL).compute()
```

Comment:
is this different to the watermask?

Reply:
Removed this in 790a5f369ac849bec3106325570aee19c58f7171